Strata/DL/Util/StringGen.lean

```diff
@@ -132,21 +120,14 @@ theorem Nat_digitChar_index: x.digitChar =
   repeat (cases x; simp; rename_i x)
   any_goals simp
 
-theorem neq_elem_of_neq_index_of_nodup (H: List.Nodup a) (Hl1: x < a.length) (Hl2: y < a.length) (Hneq: ¬ x = y): ¬ a[x]'Hl1 = a[y]'Hl2 := by
+theorem nodup_implies_injective (H: List.Nodup a) (Hl1: x < a.length) (Hl2: y < a.length) (eq : a[x]'Hl1 = a[y]'Hl2) : x = y := by
```

Comment:
This is the contrapositive of the original theorem.  I found it easier to work with.